test/e2e/specs/editor/various/scheduling.spec.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+// The `timezone` setting exposed via REST API only accepts `UTC`
+// and timezone strings by location.
+const TIMEZONES = [ 'Pacific/Honolulu', 'UTC', 'Australia/Sydney' ];
+
+test.describe( 'Scheduling', () => {
+	TIMEZONES.forEach( ( timezone ) => {
```

Comment:
Nit: Could we move the loop to close to the `test` function so that we only have to get the original timezone once? We can then call `updateSiteSettings` in `beforeEach` or inside the test too.

Reply:
I'll try it, though I don't think it should affect test speed in this case.